ras/sosreport.py

```diff
@@ -170,8 +171,8 @@ def test_others(self):
         self.run_cmd("%s --batch --tmp-dir=%s -p boot,memory" %
                      (self.sos_cmd, directory_name))
 
-        if "java" not in self.run_cmd_out("%s --batch --tmp-dir=%s -p webserver | "
-                                          "grep java" % (self.sos_cmd, directory_name)):
+        if "perl" not in self.run_cmd_out("%s --batch --tmp-dir=%s -p webserver | "
+                                          "grep perl" % (self.sos_cmd, directory_name)):
```

Comment:
Although this change looks good, can we add a check for package dependency and install it explicitly if not available on the system? This will ensure the test always works.